libs/autofile/cmd/logjack.go

```diff
@@ -39,13 +39,12 @@ func main() {
 	}
 
 	// Open Group
-	group, err := auto.OpenGroup(headPath)
+	group, err := auto.OpenGroup(headPath, auto.SetHeadSizeLimit(chopSize), auto.SetTotalSizeLimit(limitSize))
```

Comment:
if these are options now, they must not have a `Set` prefix

Reply:
yes， Set* -> With* looks better?

Reply:
GroupHeadSizeLimit()
GroupTotalSizeLimit()

Reply:
done!